### PR TITLE
Run exodus privacy scanner on all PRs

### PR DESCRIPTION
**Pull request by michaelkaye**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [ ] Other :

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

**Comment by michaelkaye** [comment]
https://github.com/vector-im/element-android/runs/6884538793?check_suite_focus=true#step:5:233 - zero trackers in the release build.

**Review by bmarty — APPROVED** [review]
Nice, thanks. One merged on develop, I would be able to rebase the other PR, and finally we will remove exodus from Buildkite.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -46,8 +46,9 @@ jobs:
   release:
     name: Build unsigned GPlay APKs
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main'
-    # Only runs on main, no concurrency.
+    concurrency:
+      group: ${{ github.ref == 'refs/head/main' && format('integration-tests-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('integration-tests-develop-{0}', github.sha) || format('build-debug-{0}', github.ref)  }}
```
[the remark this answers is not in the document]
Comment (michaelkaye) [reply .github/workflows/build.yml:50]: Was just pulling it from the other build - i need to make sure this runs on the right things before we merge; yes

**Comment by michaelkaye** [comment]
(Rebase is just to clean up commit history due to lots of tweaks)

File: .github/workflows/build.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (michaelkaye) [reply .github/workflows/build.yml]: Have changed this to be named more appropriate too.

**Pull request merged**